src/vellum/workflows/nodes/displayable/tool_calling_node/utils.py

```diff
@@ -419,26 +419,33 @@ def create_port_condition(fn_name):
                     "name"
                 ].equals(fn_name)
             )
+            # First port should be on_if, subsequent ports should be on_elif
+            return Port.on_if(condition) if is_first else Port.on_elif(condition)
 
+        is_first_port = True
```

Comment:
[nit] we can avoid this boolean tracking by maping functions -> conditions, then conditions -> ports